modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -594,7 +594,7 @@ public void testAddObjectActionWithMoreThanOneObjectEntry()
 
 		_assertWebhookObjectAction(
 			"Peter", ObjectActionTriggerConstants.KEY_ON_AFTER_ADD, null,
-			WorkflowConstants.STATUS_DRAFT);
+			WorkflowConstants.STATUS_APPROVED);
```

Comment:
Because now when we add an entry (non draft) it is persisted directly to "approved" and thus triggers the action having this status. Before it would always be persisted as "draft" first.